src/glob/match.zig

```diff
@@ -139,6 +139,7 @@ pub fn match(_: Allocator, glob: []const u8, path: []const u8) MatchResult {
     // TODO: consider just returning a bool
     // return matched != negated;
     if (negated) {
+        // FIXME(@DonIsaac): This looks backwards to me
```

Comment:
maybe better names are `no_match_due_to_negation` and `match_due_to_negation`

Reply:
I agree, but I don't plan on modifying glob code in this PR